### DISCUSSION TITLE
Add Roku Pay subscription

### DISCUSSION THREAD

**Discussion by matrixebiz** [rrirower/DP2SceneGraphWizard#53] [General]
Hello, are you able to add Roku Pay subscription based code in the Wizard so the user has to sign up to a monthly subscription for a set amount?

**Comment by rrirower** [comment]
Please follow this project for future updates. That code is currently in development.

**Comment by matrixebiz** [comment]
Ok, excellent, where do you normally post the list of your latest updates?

**Reply by rrirower** [reply]
If you 'star' or 'watch' this project, you should get update notifications.

**Comment by matrixebiz** [comment]
Ok, done. Starred and Watched releases :) Thx
